src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars__star {
+  margin-right: 4px;
+  background-image: url(../images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 :nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  border-radius: 0.5px;
```

Comment:
there should be `background-image` which set the active star
I guess border-radius is redundant at all

Reply:
Also, it doesn't work at all:
![image](https://github.com/mate-academy/layout_product-cards/assets/86923089/bb9a6a3d-5657-4f81-865d-e49dcdf89eb1)